src/main/java/si/slotex/nlp/gui/managers/RemoteApiManager.java

```diff
@@ -8,7 +8,7 @@
 @Service
 public class RemoteApiManager {
 
-    private Logger logger = Logger.getLogger(RemoteApiManager.class);
```

Comment:
Why is this logger set as public? Shouldn't it be private, since loggers are normally only used in the current class.